src/time.rs

```diff
@@ -256,3 +257,46 @@ pub fn clock_getcpuclockid(pid: Pid) -> Result<ClockId> {
         Err(Error::Sys(Errno::from_i32(ret)))
     }
 }
+
+bitflags! {
+    /// Flags that are used for arming the timer.
+    pub struct ClockNanosleepFlags: libc::c_int {
+        const TIMER_ABSTIME = libc::TIMER_ABSTIME;
+    }
+}
```

Comment:
We should use the `libc_flags` macro instead:

```suggestion
libc_bitflags! {
    /// Flags that are used for arming the timer.
    pub struct ClockNanosleepFlags: libc::c_int {
        TIMER_ABSTIME;
    }
}
```

Reply:
And, adding a doc comment for this flag would be great:)